libs/zephyr-agent/src/lib/auth/login.ts

```diff
@@ -132,11 +173,25 @@ function fallbackManualLogin(url: string): void {
   logFn('', `${blue('⏳')} Waiting for you to complete authentication in browser...`);
 }
 
-/** Opens the given URL in the default browser. */
+// Track if a browser window is already opened for auth
+let isAuthBrowserOpened = false;
```

Comment:
Will we ever have two builds running at the same time in the same nodejs process? I don't like the idea of local-but-global state.